### ISSUE TITLE
`RUN apt-get install sudo` throws sudo warning

### ISSUE THREAD

**Issue by simison** [AtomLinter/linter-docker#61]
This:
```
RUN apt-get install sudo
```

Throws unexpected warning:

> sudo(8) usage found

Noticed [via this Dockerfile](https://github.com/Automattic/jetpack/blob/bacbb8a6087e48943611e89bf64735a393c2adc0/docker/Dockerfile#L7-L36)

**Comment by Arcanemagus** [comment]
You'll want to report this to [`dockerlint`](https://github.com/RedCoolBeans/dockerlint) if it isn't already, this project is just a wrapper around that.
